test/decl/var/behaviors.swift

```diff
@@ -0,0 +1,193 @@
+// RUN: %target-parse-verify-swift -module-name Main
+
+protocol behavior {
+  associatedtype Value
```

Comment:
Yeah, those both ought to work. My hacked-up hand-rolled conformances probably won't handle new associated type requirements yet if constraints on `Value` introduce them.